kaldb/src/test/java/com/slack/kaldb/server/KaldbTest.java

```diff
@@ -134,12 +133,12 @@ private KaldbConfigs.KaldbConfig makeKaldbConfig(
       int maxOffsetDelay) {
     return KaldbConfigUtil.makeKaldbConfig(
         "localhost:" + kafkaServer.getBroker().getKafkaPort().get(),
-        indexPort,
+        port,
         kafkaTopic,
         kafkaPartition,
         clientName,
         TEST_S3_BUCKET,
-        queryPort,
+        port + 1,
```

Comment:
Looks like this was inadvertently reverted in a rebase to the previous logic. Should continue to be explicitly defined.